package.json

```diff
@@ -129,6 +130,7 @@
     "stylelint-config-recommended-scss": "^14.0.0",
     "stylelint-scss": "^6.2.1",
     "ts-jest": "^29.1.2",
-    "typescript": "^5.4.2"
+    "typescript": "^5.4.2",
+    "yaml": "^2.4.1"
```

Comment:
This is another dependency at build time, but not only is it widely used, it's [by our colleague Eemeli](https://github.com/eemeli/yaml).